src/client/__tests__/e2e/pages/tile.component.ts

```diff
@@ -94,4 +97,42 @@ export class TileComponent {
     await textElement.clear()
     await textElement.sendKeys(notional)
   }
+
+  async initiateRFQ(initiateRFQ: string) {
+    const labelTextStreaming = this.tradeType[initiateRFQ].labelTextStreamingUnavailable
+    expect(labelTextStreaming.getText()).toEqual('STREAMING PRICE UNAVAILABLE')
+    const buttonRFQ = this.tradeType[initiateRFQ].buttonInitiateRFQ
+    await waitForElementToBeClickable(this.browser, buttonRFQ)
+    await buttonRFQ.click()
+    const buttonReject = this.tradeType[initiateRFQ].buttonReject
+    await waitForElementToBeClickable(this.browser, buttonReject)
+    await buttonReject.click()
+    const labelTextExpired = this.tradeType[initiateRFQ].labelTextTradeExpired
+    expect(labelTextExpired.getText()).toEqual('EXPIRED')
+    // To avoid click interception between the elements
+    await wait(800)
+    const buttonRequote = this.tradeType[initiateRFQ].buttonInitiateRFQ
+    await waitForElementToBeClickable(this.browser, buttonRequote)
+    await buttonRequote.click()
+    // Wait for button reload to appear after requote
+    await wait(10500)
+    const buttonReload = this.tradeType[initiateRFQ].buttonReload
+    await waitForElementToBeClickable(this.browser, buttonReload)
+    await buttonReload.click()
+  }
+
+  async NZDToUSDRFQ(currencyTrade: string) {
```

Comment:
same here.. the method name suggests we are dealing with and Rfq for NZD/USD, why does it take that `currencyTrade` param?